cmd/certificates-controller/OWNERS

```diff
@@ -0,0 +1,67 @@
+approvers:
```

Comment:
Copied verbatim from `cmd/kube-controller-manager/OWNERS`, since the code and responsibility has a lot in common with `kube-controller-manager`. I could also bootstrap it from `pkg/controller/certificates/OWNERS` instead, or omit it entirely and fall back on `cmd/OWNERS`. I'm definitely open to suggestions on the best route to bootstrap this.